gto-support-api/build.gradle

```diff
@@ -2,6 +2,6 @@ dependencies {
     compile project(':gto-support-core')
     compile project(':gto-support-api-base')
 
-    compile 'com.google.guava:guava:18.0'
+    compile 'com.google.guava:guava:' + deps.guava
```

Comment:
allows configuration of dependency version numbers to be central instead of sprinkled throughout the code.